ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/cesium.hbs

```diff
@@ -23,8 +23,8 @@
 <div class="mapInfo">
     <div class="info-feature"></div>
     <div class="info-coordinates">
-        <span class="coordinate-lat"></span>
-        <span class="coordinate-lon"></span>
+        <span></span>
+        <span></span>
```

Comment:
✏️ 	Do we still need these spans?

Reply:
Not 100% sure. Cesium might be using them for something. I just caught the redundant class. No one was referencing that.